pkg/proxy/request.go

```diff
@@ -228,18 +242,22 @@ func checkCsvFile(s string) error {
 }
 
 func getTestFile(r *http.Request) (string, error) {
-	content, _, err := getFileFromHTTP(r, testFile)
-	return content, err
+	content, fileType, err := getFileFromHTTP(r, testFile)
+	if err != nil {
+		return "", err
+	}
+
+	for _, f := range testFileFormats {
+		if fileType == f {
+			return content, err
+		}
+	}
```

Comment:
this could be a map[string]bool too, to avoid the loop/if :
```golang
var testFileFormats = map[string]bool{
    "jmx":  true,
    "py":   true,
    "json": true,
    "toml": true
}
```


```golang
if accepted := testFileFormats[fileType]; !accepted {
    return "", ErrWrongFileFormat
}
return content, nil
```